### PR TITLE
Github Actions workflow

### PR DESCRIPTION
**Pull request by aramboi**
Run tests in Github Actions instead of Travis.

### CODE REVIEW

**Comment by aramboi** [comment]
Github actions seems a bit faster (1m 50s) than Travis CI (2 min 11 sec), which makes sense, as it might be using the same infrastructure.

File: .github/workflows/pythonapp.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: EuroPython website
+
+on:
+  push:
+    branches:
+    - master
+    - dev/ep*
```
Comment (umgelurgel) [inline .github/workflows/pythonapp.yml]: Could you add `ep20*` to the branches and pull request triggers? We're going to move to `ep2020` branch soon - and probably just keep the `ep20XX` branches in the future, I don't think we should keep a separate `dev/20xx` branch after stopping using the current one.

**Review by umgelurgel — CHANGES_REQUESTED** [review]
Looks great - thanks for fixing this! Left one small comment.

**Review by umgelurgel — APPROVED** [review]
This is awesome! Thanks for setting this up! :shipit:

**Pull request merged**